src/elements/header-bottom-toolbar.html

```diff
@@ -94,10 +94,13 @@
             </template>
           </template>
         </template>
-        <paper-tab class="nav-item" day="my-schedule" hidden$="[[!user.signedIn]]" link>
-          <a href$="[[_addQueryParams('my-schedule', queryParams)]]" layout vertical center-center>{$ mySchedule.title
-            $}</a>
-        </paper-tab>
+        <template is="dom-if" if="[[user.signedIn]]">
```

Comment:
Switching this to a `dom-if` meant the anchor doesn't exist when it shouldn't be shown. Previously, it still existed and was showing up in focusable elements. https://dequeuniversity.com/rules/axe/4.7/aria-hidden-focus